### DISCUSSION TITLE
Creating multiple scatter plots on a single figure

### DISCUSSION THREAD

**Discussion by davidDS96** [alandefreitas/matplotplusplus#272] [Q&A]
Hello everyone,

I was wondering if it is possible to create, for example, two scatter plots (or more) displayed on one figure, preferably in a different color.

Kind regards, 
David

**Comment by alandefreitas** [comment]
See

- https://alandefreitas.github.io/matplotplusplus/plot-types/data-distribution/scatter-plot/
- https://alandefreitas.github.io/matplotplusplus/appearance/multiplot/hold/
- https://alandefreitas.github.io/matplotplusplus/appearance/multiplot/subplots/

**Reply by davidDS96** [reply]
Thank you very much! I'll consult those pages.
